### PR TITLE
Tweak: Display column-gap control conditionally on Text Editor widget [ED-13474]

### PR DESCRIPTION
**Pull request by rami-elementor**

### CODE REVIEW

File: includes/widgets/text-editor.php [hunk 1]
```diff
@@ -126,17 +126,25 @@ protected function register_controls() {
...
+					'1' => '1',
+					'2' => '2',
+					'3' => '3',
+					'4' => '4',
+					'5' => '5',
+					'6' => '6',
+					'7' => '7',
+					'8' => '8',
+					'9' => '9',
+					'10' => '10',
```
Comment (rami-elementor) [inline includes/widgets/text-editor.php:146]: I'm considering to add translation string for languages like Arabic that have their own numbers.

File: includes/widgets/text-editor.php [hunk 2]
```diff
@@ -126,17 +126,25 @@ protected function register_controls() {
...
 		$this->add_responsive_control(
 			'text_columns',
 			[
 				'label' => esc_html__( 'Columns', 'elementor' ),
 				'type' => Controls_Manager::SELECT,
 				'separator' => 'before',
-				'options' => $text_columns,
+				'options' => [
+					'' => esc_html__( 'Default', 'elementor' ),
+					'1' => esc_html__( '1', 'elementor' ),
```
Comment (arielk) [inline includes/widgets/text-editor.php:137]: Why we need to translate numbers?

File: includes/widgets/text-editor.php [hunk 3]
```diff
@@ -126,17 +126,25 @@ protected function register_controls() {
...
+					'2' => esc_html__( '2', 'elementor' ),
+					'3' => esc_html__( '3', 'elementor' ),
+					'4' => esc_html__( '4', 'elementor' ),
+					'5' => esc_html__( '5', 'elementor' ),
```
Comment (arielk) [inline includes/widgets/text-editor.php:141]: Why this method it better then before?

File: includes/widgets/text-editor.php (hunk 2 above)
Comment (rami-elementor) [reply includes/widgets/text-editor.php:137]: @arielk see my comment above.

Languages like Arabic and Persian that have their own numbers.

File: includes/widgets/text-editor.php (hunk 3 above)
Comment (rami-elementor) [reply includes/widgets/text-editor.php:141]: Same as above...

**Review by TzviRabinovitch — APPROVED** [review]

**Review by ronenboxer — APPROVED** [review]

File: includes/widgets/text-editor.php (hunk 2 above)
Comment (arielk) [reply includes/widgets/text-editor.php:137]: @rami-elementor Does WordPress, WooCommerce or Yoast use this method as well?

**Review by arielk — APPROVED** [review]

**Pull request merged**